### DISCUSSION TITLE
Does multipar can repaire .par2 itself？

### DISCUSSION THREAD

**Discussion by ryhvip** [Yutaka-Sawada/MultiPar#101] [Q&A]
I'm just wondering if .par2 is corrupred a little，is it could repaire it self？Because i found that the WinRAR can  do this，however MultiPar only verifies the .par2 is broken，MuitiPar can repair the original files but can't repair .par2 itself。Is it because I won't use it？Thank you so much！

**Comment by Yutaka-Sawada** [comment]
> I'm just wondering if .par2 is corrupred a little，is it could repaire it self？

No, it cannot repair PAR2 files. If you want to get complete PAR2 file, you need to recreate the PAR2 file again. There is "Extra" button on the right bottom of Verify window. By setting "First recovery block number" and "Number of Recovery blocks", you can make favorite PAR2 file for the set of source files. To make one PAR2 file, select "Sizing scheme" to "All the same size", and set "Number of Recovery files" to 1.

> Is it because I won't use it？

Even when some PAR2 files are broken, it can use them to repair source files. While there is enough redundancy, no need to repair (recreate) broken PAR2 files. If you want to keep more recovery blocks for future damage, you may create additional PAR2 files. That is the usage of "Extra" button.

**Reply by ryhvip** [reply]
Thank you for your patience answer, I have learned it.
